lib/soql/soql.js

```diff
@@ -13,7 +14,11 @@ class SoQL {
    * TODO: this is necessary, but is this sufficient?
    */
   _launderName(name) {
-    return changeCase.snakeCase(name);
+    name = changeCase.snakeCase(name);
+    if(!_.isNaN(parseInt(name[0]))) {
+      name = '_' + name;
+    }
+    return name;
   }
```

Comment:
I believe that its so, but I'm just curious about why soda fountain doesn't approve of column names starting with numbers?